examples/usdc-transfer-app/package.json

```diff
@@ -0,0 +1,32 @@
+{
+	"name": "usdc-transfer-app",
+	"version": "0.1.0",
+	"type": "module",
+	"private": true,
+	"license": "Apache-2.0",
+	"scripts": {
+		"dev": "next dev",
+		"build": "next build",
+		"start": "next start",
+		"lint": "next lint"
+	},
+	"dependencies": {
+		"@mysten/dapp-kit": "workspace:*",
+		"@mysten/sui": "^1.11.0",
+		"@mysten/wallet-kit": "^0.8.6",
```

Comment:
(this is the cause of the build failure): Ideally you should use all deps from workspace, or pin them together in a "play-well" fashion.

Reply:
Thanks! I forgot I changed this because I was having issues with `pnpm install` for some weird reason. Would have never recalled.